src/main/java/com/kodilla/royal/game/of/ur/Dice.java

```diff
@@ -0,0 +1,30 @@
+package com.kodilla.royal.game.of.ur;
+
+import java.util.Random;
+
+public class Dice {
+
+    private final Random diceRoller;
```

Comment:
kostka nie powinna miec dice roller, to raczej dice roller powinien miec kostke

Reply:
Nazwa tego pola jest myląca - nie jest to diceRoller, tylko obiekt generujący liczby psudolosowe.
Zmieniłem jego nazwę na 'rollResultGenerator', która wydaje mi się bardziej odpowiednia.
Zacząłem się też zastanawiać nad klasą DIceRoller, ale tu mam problem, bo obiekt Dice istnieje tylko po to, żeby wywoływać na nim metodę roll(). Podobnie byłoby z obiektem klasy DiceRoller - jego jedynym zadaniem byłoby rzucenie kostką. Jeśli przeniósłbym metodę roll() do klasy DiceRoller, klasa Dice stałaby się zbędna. Mógłbym też zostawić klasę Dice bez zmian i przekazywać kostkę do klasy DiceRoller w celu wykonania rzutu, ale w tym wypadku wydaje mi się, że byłby to tylko dodatkowy pośrednik, który niczemu by nie służył.
Mógłbym też po prostu zmienić nazwę klasy Dice na DiceRoller, ale nie wiem, czy jest sens.